plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -1088,11 +1094,17 @@ def disableTimer(self) -> None:
             self._change_timer.timeout.disconnect(self.slice)
 
     def _onPreferencesChanged(self, preference: str) -> None:
-        if preference != "general/auto_slice":
+        if preference != "general/auto_slice" and preference != "info/send_engine_crash":
             return
-        auto_slice = self.determineAutoSlicing()
-        if auto_slice:
-            self._change_timer.start()
+        if preference == "general/auto_slice":
+            auto_slice = self.determineAutoSlicing()
+            if auto_slice:
+                self._change_timer.start()
+        elif preference == "info/send_engine_crash":
```

Comment:
Why is this in a `elif`? We don't want to send sentry data when auto slice is enabled?

Reply:
This is about handing a change in preference. If the preference that is being changed is anything but "info/send_engine_crash" we don't want to change this particular behavior.

Reply:
ah, right. read too fast